src/MendelIHT_utilities.jl

```diff
@@ -168,4 +183,40 @@ function std_reciprocal(A::SnpArray, mean_vec::Vector{Float64})
     return std_vector
 end
 
+""" Projects the point y onto the set with at most m active groups and at most 
+n active predictors per group. The variable group encodes group membership. Currently
+assumes there are no unknown group membership."""
+function doubly_sparse_projection(y::Vector{Float64}, group::Vector{Int64}, m::Int64, n::Int64)
```

Comment:
is this the group sparse projection?

since this is prototype code, I won't fuss too much about it. but for final code, you should

1. not type on `Float64` alone; use something general like `T <: AbstractFloat` or `T <: Float` (using IHT alias for `Float`)
2. splitting this function into a mutating function that operates on `y` using code from lines `191:218` and a nonmutating function that operates on `x = copy(y)`

also consider making the function name verby, something like

`project_group_sparse!`

Reply:
I fixed (1) but I'm going to leave (2) until after I finish my poster